opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,415 @@
+# Copyright 2017, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import os
+import time
+import platform
+from . import base
+from google.cloud import monitoring_v3
+from opencensus.stats import view_data
+from opencensus.stats import aggregation
+from opencensus.stats import aggregation_data
+from opencensus.stats import stats
+from opencensus.stats import measure
+
+MAX_TIME_SERIES_PER_UPLOAD = 200
+OPENCENSUS_TASK_DESCRIPTION = "Opencensus task identifier"
+DEFAULT_DISPLAY_NAME_PREFIX = "OpenCensus"
+ERROR_BLANK_PROJECT_ID = "expecting a non-blank ProjectID"
+CONS_NAME = "name"
+CONS_TIME_SERIES = "timeseries"
+
+class Options(object):
```

Comment:
if `Options` is to be an immutable class you can just define it as a `namedtuple`.
it acts is a data-class without the boilerplate.
https://pymotw.com/2/collections/namedtuple.html